tests/sp2n/Test_project_on_Sp.cc

```diff
@@ -17,27 +17,32 @@ int main (int argc, char **argv)
     
     LatticeGaugeField Umu(&Grid);
     LatticeColourMatrixD U(&Grid);
+    LatticeColourMatrixD Up(&Grid);
     LatticeColourMatrixD aux(&Grid);
     LatticeColourMatrixD identity(&Grid);
     
+    // Will test resimplectification-related functionalities (from ProjectOnGaugeGroup, ProjectOnSpGroup, ProjectGn) and projection on the algebra (from ProjectSp2nAlgebra)
+    // we work with matrices with positive determinant so detU = 1 even if in principle ProjectOnGaugeGroup and ProjectOnSpGroup allow for detU=-1
```

Comment:
Should we also check negative determinant? Seems like an easy thing to add and a foolish thing to trip over later.